goldlib/gall/gdefs.h

```diff
@@ -346,6 +346,14 @@ template <class T> inline bool make_bool_not(T a)
 #define MinV minimum_of_two
 #define MaxV maximum_of_two
 
+/*  ------------------------------------------------------------------ */
+/* rigister deprecated since C++11                                     */
+
+#if __cplusplus > 199711L
+#define register
```

Comment:
I'd better just remove that keyword in all places in golded codebase. I don't believe it improves performance at all.

Reply:
Yeah, will do